service/src/main/java/com/ge/predix/acs/privilege/management/dao/GraphGenericRepository.java

```diff
@@ -69,10 +69,10 @@ public void deleteInBatch(final Iterable<E> entities) {
     @Override
     public List<E> findAll() {
         try {
-            return this.graph.traversal().V().has(ZONE_ID_KEY).has(getEntityIdKey()).toList().stream()
+            return this.graphTraversal.V().has(ZONE_ID_KEY).has(getEntityIdKey()).toList().stream()
                     .map(this::vertexToEntity).collect(Collectors.toList());
         } finally {
-            this.graph.tx().commit();
+            this.graphTraversal.tx().commit();
```

Comment:
I spoke with Sanjeev and Anubhav, and they want to keep it for now for historical reasons.